src/Datadog.Trace.Tools.Runner/Utils.cs

```diff
@@ -0,0 +1,209 @@
+            //  C:\Users\[user]\.dotnet\tools\.store\datadog.trace.tools.runner\1.19.3\datadog.trace.tools.runner\1.19.3\tools\netcoreapp3.1\any
+            //  C:\Users\[user]\.dotnet\tools\.store\datadog.trace.tools.runner\1.19.3\datadog.trace.tools.runner\1.19.3\tools\netcoreapp2.1\any
+            // And the Home folder is:
+            //  C:\Users\[user]\.dotnet\tools\.store\datadog.trace.tools.runner\1.19.3\datadog.trace.tools.runner\1.19.3\home
+            // So we have to go up 3 folders.
+
+            string tracerHome = EnsureFolder(options.TracerHomeFolder ?? Path.Combine(runnerFolder, "..", "..", "..", "home"));
+            string tracerMsBuild = EnsureFile(Path.Combine(tracerHome, "netstandard2.0", "Datadog.Trace.MSBuild.dll"));
+            string tracerIntegrations = EnsureFile(Path.Combine(tracerHome, "integrations.json"));
+            string tracerProfiler32 = string.Empty;
+            string tracerProfiler64 = string.Empty;
+
+            if (platform == Platform.Windows)
+            {
+                tracerProfiler32 = EnsureFile(Path.Combine(tracerHome, "win-x86", "Datadog.Trace.ClrProfiler.Native.dll"));
+                tracerProfiler64 = EnsureFile(Path.Combine(tracerHome, "win-x64", "Datadog.Trace.ClrProfiler.Native.dll"));
+            }
+            else if (platform == Platform.Linux)
+            {
+                tracerProfiler64 = EnsureFile(Path.Combine(tracerHome, "linux-x64", "Datadog.Trace.ClrProfiler.Native.so"));
+            }
+
+            var envVars = new Dictionary<string, string>
+            {
+                ["DD_DOTNET_TRACER_HOME"] = tracerHome,
+                ["DD_DOTNET_TRACER_MSBUILD"] = tracerMsBuild,
+                ["DD_INTEGRATIONS"] = tracerIntegrations,
+                ["CORECLR_ENABLE_PROFILING"] = "1",
+                ["CORECLR_PROFILER"] = PROFILERID,
+                ["CORECLR_PROFILER_PATH_32"] = tracerProfiler32,
+                ["CORECLR_PROFILER_PATH_64"] = tracerProfiler64,
+                ["COR_ENABLE_PROFILING"] = "1",
+                ["COR_PROFILER"] = PROFILERID,
+                ["COR_PROFILER_PATH_32"] = tracerProfiler32,
+                ["COR_PROFILER_PATH_64"] = tracerProfiler64,
+            };
+
+            if (!string.IsNullOrWhiteSpace(options.Environment))
+            {
+                envVars["DD_ENV"] = options.Environment;
+            }
+
+            if (!string.IsNullOrWhiteSpace(options.Service))
+            {
+                envVars["DD_SERVICE"] = options.Service;
+            }
+
+            if (!string.IsNullOrWhiteSpace(options.Version))
+            {
+                envVars["DD_VERSION"] = options.Version;
+            }
+
+            if (!string.IsNullOrWhiteSpace(options.AgentUrl))
+            {
+                envVars["DD_TRACE_AGENT_URL"] = options.AgentUrl;
+            }
+
+            return envVars;
+        }
+
+        public static string EnsureFolder(string folderName)
```

Comment:
Consider renaming to `EnsureDirectoryExists()` or simply `DirectoryExists()`.

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/962/commits/cef64035cd2188cb8199f3e8e71cb43c76cfe128